### ISSUE TITLE
TextField: Error text is not highlighted in High contrast mode

### ISSUE THREAD

**Issue by nipunja** [OfficeDev/office-ui-fabric-react#792]
### Bug Report

- __Package version(s)__: (fill this out)
- __Browser and OS versions__: (fill this out if relevant)

#### Describe the issue:
Error text is coming in same color & font as the normal text. So not able distinguish error in high contrast.

![image](https://cloud.githubusercontent.com/assets/11870255/21847979/a4b1313e-d824-11e6-828d-5f5cc3c3657f.png)

#### Actual behavior:
Not able to distinguish error context from the rest in high contrast.

#### Expected behavior:
Error region should be distinguishable.

### If applicable, please provide a codepen repro:
https://fabricreact.azurewebsites.net/fabric-react/master/#/examples/textfield (see in high contrast)

**Comment by cschlechty** [comment]
We should just add a glyph before the error (like !)

**Comment by Adity-Agarwal** [comment]
@cschlechty Any updates on this? Has the glyph been added?
